packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1795,6 +1800,14 @@ export function pingSuspendedRoot(
     return;
   }
 
+  if (suspendedTime === Offscreen) {
```

Comment:
Why special case this at all? Should this be an invariant?

Reply:
This should restart the render if a ping is in the middle of an offscreen render.